src/Event/Http/HttpResponse.php

```diff
@@ -20,48 +16,40 @@ final class HttpResponse
     /** @var string */
     private $body;
 
-    public function __construct(string $body, array $headers, int $statusCode = 200)
+    public function __construct(string $body, array $headers = [], int $statusCode = 200)
     {
         $this->body = $body;
         $this->headers = $headers;
         $this->statusCode = $statusCode;
     }
 
-    public static function fromPsr7Response(ResponseInterface $response): self
```

Comment:
The logic related to PSR-7 was moved to the Psr7Bridge class.

The logic related to HTTP responses was kept in this class below (it will then apply to all HTTP responses).